AK/SinglyLinkedList.h

```diff
@@ -81,9 +81,15 @@ class SinglyLinkedList {
 public:
     SinglyLinkedList() = default;
     SinglyLinkedList(const SinglyLinkedList& other) = delete;
-    SinglyLinkedList(SinglyLinkedList&&) = default;
+    SinglyLinkedList(SinglyLinkedList&& other)
+        : m_head { other.m_head }
+        , m_tail { other.m_tail }
```

Comment:
We don't use brace initializers in constructor initializer lists (as in, this should be `m_head(other.m_head)` instead